src/backend/distributed/commands/call.c

```diff
@@ -96,13 +96,12 @@ CallFuncExprRemotely(CallStmt *callStmt, DistObjectCacheEntry *procedure,
 	CitusTableCacheEntry *distTable = GetCitusTableCacheEntry(colocatedRelationId);
 	Var *partitionColumn = distTable->partitionColumn;
 	bool colocatedWithReferenceTable = false;
-	if (IsCitusTableTypeCacheEntry(distTable, REFERENCE_TABLE))
+	if (IsCitusTableTypeCacheEntry(distTable, CITUS_TABLE_WITH_NO_DIST_KEY))
```

Comment:
Does that change make sense ?

Reply:
I think we should punt on this change for now, because we have a lot of reference table usage: https://github.com/citusdata/citus/compare/single-placement-table/master-cache-entry-rebased...add_funct_call_delegation?expand=1


I'd suggest we make this back as REFERENCE_TABLE and add `Support function call delegation` to the todo list. Maybe share  this super basic change that potentially allows it.